src/components/common/OfficeAddresses.vue

```diff
@@ -466,188 +408,130 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
+    address.actions = address.actions.filter(action => action !== val)
   }
 
-  /**
-   * Initialize and assign base states of Address Data
-   *
-   */
-  private initAddresses (): void {
-    // If loading loading from a draft filing
-    if (this.addresses) {
-      this.addresses.registeredOffice && this.assignRegisteredAddresses(this.addresses.registeredOffice)
-      this.addresses.recordsOffice && this.assignRecordAddresses(this.addresses.recordsOffice)
-    } else {
-      // If new filing load from addresses from store
-      this.registeredAddress && this.assignRegisteredAddresses(this.registeredAddress)
-      this.recordsAddress && this.assignRecordAddresses(this.recordsAddress)
-    }
-
-    // emit address data back up so that parent data has data (needed for AR filing specifically)
-    this.emitAddresses()
+  /** When form valid state changes, emit event up to parent. */
+  @Watch('formValid')
+  private onFormValidChanged (formValid: boolean): void {
+    this.emitValid(formValid)
   }
 
-  /**
-   * Assign the Registered address objects actions and omit the addressType
-   *
-   * @param addressBase The base address object to be parsed and assigned to its Office Address class property
-   */
-  private assignRegisteredAddresses (addressBase: BaseAddressObjIF): void {
-    // Assign Delivery Address
-    const deliveryAddress = addressBase.deliveryAddress
-    if (deliveryAddress) {
-      deliveryAddress.actions = []
-      this.deliveryAddressOriginal = { ...deliveryAddress }
-      if (isEmpty(this.deliveryAddress)) {
-        this.deliveryAddress = { ...deliveryAddress }
-      }
-    } else {
-      // eslint-disable-next-line no-console
-      console.log('invalid Delivery Address =', addressBase)
-    }
+  /** Emits the valid state of the addresses. */
+  @Emit('valid')
+  private emitValid (val: boolean): void {}
 
-    // Assign Mailing Address
-    const mailingAddress = addressBase.mailingAddress
-    if (mailingAddress) {
-      mailingAddress.actions = []
-      this.mailingAddressOriginal = { ...mailingAddress }
-      if (isEmpty(this.mailingAddress)) {
-        this.mailingAddress = { ...mailingAddress }
-      }
-    } else {
-      // eslint-disable-next-line no-console
-      console.log('invalid Mailing Address =', addressBase)
-    }
+  /** Emits the modified state of the addresses. */
+  @Emit('modified')
+  private emitModified (): boolean {
+    return this.anyModified
   }
 
-  /**
-   * Assign the Records address objects actions and omit the addressType
-   *
-   * @param addressBase The base address object to be parsed and assigned to its Office Address class property
-   */
-  private assignRecordAddresses (addressBase: BaseAddressObjIF): void {
-  // Assign Delivery Address
-    const deliveryAddress = addressBase.deliveryAddress
-    if (deliveryAddress) {
-      deliveryAddress.actions = []
-      this.recDeliveryAddressOriginal = { ...deliveryAddress }
```

Comment:
This code was originally part of the addresses initialization.

You'll notice here that any "actions" are cleared (for easier comparison in this component) but I thought this was worth keeping and I simply updated the comparison code to account for it.

You'll also notice that the original addresses are set here, which is one of the bugs I fixed -- when the user resumes a draft filing, we should NOT assign the original addresses from the draft data -- we should instead load the current addresses from the API.